include/LightGBM/config.h

```diff
@@ -392,6 +392,7 @@ struct Config {
   // desc = will stop training if one metric of one validation data doesn't improve in last ``early_stopping_round`` rounds
   // desc = ``<= 0`` means disable
   // desc = can be used to speed up training
+  // desc = *New in 4.4.0*
```

Comment:
This should be below (on early_stopping_min_delta)

Reply:
thank you for catching this! I just pushed a commit fixing it.